test/bench_sub_async.c

```diff
@@ -222,6 +230,7 @@ static void _benchMatrix(threadConfig *threadsVector, int lent, int *subsVector,
         }
```

Comment:
This is so that each sub gets the same number of messages, since we average/best/worst across subs. The output includes the actual total number of messages received.

Reply:
But again, we have:
```
int numPubMessages = NMessages / numSubs;
...
printf("\t{\"subs\":%d, \"threads\":%d, \"messages\":%d, \"best\":%d, \"average\":%d, \"worst\":%d}%s\n",
                   numSubs, env->threads.useGlobalDelivery ? env->threads.max : 0, numPubMessages * numSubs, best, average, worst, comma);
```
This is for a given `_benchMatrix` run. So we divide the number of messages (NMessages) by the number of subs, but then print out that number * numSubs. So to me, it looks like using NMessages should be used?

Reply:
Ok.